flake.nix

```diff
@@ -21,5 +21,14 @@
         formatter = pkgs.nixpkgs-fmt;
 
         devShells.default = import ./nix/shell.nix { inherit pkgs inputs; };
-      });
+      }) 
+    // rec {
+      overlays = {
+        default = overlays.yazi;
+        yazi = final: prev: {
+          yazi = final.callPackage ./nix/yazi.nix { };
+        };
+      };
+      overlay = overlays.default;
```

Comment:
if you dont need it, let's remove the "overlay" since it's deprecated.

Reply:
Didn't know there was a deprecation. I though it was just a unofficial semi-common community practice. But it looks like you're right: https://github.com/NixOS/nix/blob/master/doc/manual/src/release-notes/rl-2.7.md